docs/source/overview/using_dsh.md

```diff
@@ -0,0 +1,82 @@
+(using_dsh)=
+
+# Before using the Data Safe Haven
+
+This page contains some important considerations you must take before using the Data Safe Haven.
+Where appropriate, there are links to external resources, including policies and processes used at The Turing.
+
+```{warning}
+Use of a Data Safe Haven is not by itself sufficient to guarantee the security of your data! It must be paired with appropriate information governance requirements and user agreements.
+```
+
+```{warning}
+Each organisation deploying their own instance of the Data Safe Haven is responsible for verifying their Data Safe Haven instance is deployed as expected and that the deployed configuration effectively supports their own information governance policies and processes.
+
+Each organisation deploying their own instance of the Data Safe Haven is responsible for verifying that the instance is configured as expected. The organisation is also reponsible for confirming that the deployed configuration is appropriate for their purposes and effectively supports their own information governance policies and processes. We provide the Data Safe Haven code and material on an ‘as is’ basis without warranties of any kind and you use the code and supporting materials at your own cost and risk.
+```
+
+```{tip}
+In terms of the [Five Safes framework](https://ukdataservice.ac.uk/help/secure-lab/what-is-the-five-safes-framework/) the Data Safe Haven is aiming to be a Safe Setting.
+```
+
+## What is needed to run a Data Safe Haven?
+
+The code of this project is not on its own sufficient to operate a secure environment for research on sensitive data.
+In fact, _any_ functional TRE is not just code and infrastructure, but also people, policies, and processes.
+This project provides code to deploy a TRE with a particular architecture and the documentation gives instructions and advice for operating an instance.
+Every group deploying the Data Safe Haven will need to provide the rest including,
+
+- Information governance processes
+    - How to approve data ingress and egress
+    - Classifying work into Data Safe Haven tiers
+- Mapping internal roles or people to Data Safe Haven roles
+- Staffing essential roles such as system and programme managers
+- Data security incident handling procedures
+- Financial planning
+- Supporting infrastructure such as
+    - Communication channels
+    - Domains and DNS
+    - Secure methods to share SAS tokens
+    - Programme management tools
+
```

Comment:
Could we bring in a SATRE reference here too?

e.g. For a comprehensive guide to the technical and organisational infrastructure required to fully support running of a DSH, see the [Standard Architecture for Trusted Research Environments (SATRE)](https://satre-specification.readthedocs.io/en/stable/)